### DISCUSSION TITLE
Dgus hmi firmware

### DISCUSSION THREAD

**Discussion by TheFeralEngineer** [NARUTOfzr/Neptune_3#49] [General]
What is the plan to update the dgus firmware on the original Neptune 3? With the pro being released and the plus coming out soon, it feels like the Neptune 3 is being forgotten with firmware updates. I wish to have the 2.1.1 Marlin and the UI features that the pro has on my Neptune 3. It's almost as if this machine is obsolete before being out of pre-release

**Comment by leninkster** [comment]
@TheFeralEngineer I'm afraid your fears may be true.  Quote from Elegoo in recent communications:  "_We have to say sorry to you that the Neptune 3 is out of stock now, and it has stopped producing, and will not re-stock anymore_"

If you can get your hands on the Neptune 3, it's still a great machine, and with the price drops it's got huge potential for prototyping upgrades, but from here on in The Neptune 3 Pro & Plus range appears to be the way to go.

**Reply by TheFeralEngineer** [reply]
> @TheFeralEngineer I'm afraid your fears may be true. Quote from Elegoo in recent communications: "_We have to say sorry to you that the Neptune 3 is out of stock now, and it has stopped producing, and will not re-stock anymore_"
> 
> If you can get your hands on the Neptune 3, it's still a great machine, and with the price drops it's got huge potential for prototyping upgrades, but from here on in The Neptune 3 Pro & Plus range appears to be the way to go.

I have the Pro firmware running cleanly on a neptune 3 with an MKS H43 screen. Actually, I HAD it running, now I'm using klipper, but I had a clean install of a slightly modded 2.1.1 based on the official Elegoo firmware, so updates to the latest firmware are quite easy to handle :D
